pkgs/top-level/all-packages.nix

```diff
@@ -34634,9 +34634,10 @@ with pkgs;
 
 
   inherit (callPackages ../applications/science/logic/z3 { python = python2; })
+    z3_4_11
     z3_4_8
     z3_4_7;
```

Comment:
```suggestion
```
Can we drop 4.7?

Reply:
What if someone is using it?

I'm not sure what the standard is here but I'll do this if it's what I should do.

Reply:
If you _do_ choose to remove it, add it to `aliases.nix`.

Reply:
Please do _not_ alias it. Different versions of Z3 are completely incompatible.

Reply:
`aliases.nix` is largely used to "alias" removed packages with `throw` statements providing guidance.